### DISCUSSION TITLE
新手暴毙

### DISCUSSION THREAD

**Discussion by cinema8150** [ant-design/ant-design#46101] [General]
新手暴毙，使用官方CLI（https://pro.ant.design/zh-CN/docs/getting-started#%E5%88%9D%E5%A7%8B%E5%8C%96）创建项目，vscode打开后各种文件报红，错误信息和本地环境截图里有，求指导 🙏：
![image](https://github.com/ant-design/ant-design/assets/77312323/517bf5e8-3154-4dc4-8f5b-950812fd430b)

**Comment by XueMeijing** [comment]
https://github.com/ant-design/pro-components/issues/6592

这个里面提到说是ts版本不一致的问题，可以点进去看看
